src/Azure.Functions.Cli/Helpers/WorkerRuntimeLanguageHelper.cs

```diff
@@ -42,6 +42,17 @@ public static class WorkerRuntimeLanguageHelper
             { WorkerRuntime.powershell, "powershell" }
         };
 
+        public static IDictionary<WorkerRuntime, string> WorkerToDisplayStrings => availableWorkersRuntime.Keys.ToDictionary(wr => wr, wr =>
+        {
+            switch (wr)
+            {
+                case WorkerRuntime.python:
+                    return "python (preview)";
```

Comment:
Wondering if we should have a dictionary (WorkerRuntime/string) with runtime and display strings so we could properly format them (including proper casing and runtime name, like `.NET` instead of `dotnet`)

Reply:
That is a good call. I wanted to be least invasive and did not want to add another dictionary to maintain if a new WorkerRuntime is added or changed.
But, I think it should be ok as it's all in the same place and if you think we should change the display strings.

Reply:
Yeah, I get where you were coming from, just wanted to share that thought as I think it will help us provide a better experience than just the enum `ToString()` result

Reply:
The only other thing I could think of is we will either have to
1) have an inconsistency between `--worker-runtime` and the display strings. (`--worker-runtime dotnet` vs display string being `.NET`)
2) make changes to ensure params `--worker-runtime .NET` work.

Reply:
That's a good point. I was looking at this as just plain human readable runtime names, but I guess people could use what's displayed for the parameter/setting value as well. That's actually a good argument for leaving this as is.

Reply:
Yeah, that makes sense. Sounds good. I could open a separate PR that changes the display strings and accepts both, the display string and what we have now to be passed as `--worker-runtime`

Reply:
Sounds good